### DISCUSSION TITLE
[rehype-autolink-headings] Customize generated href.

### DISCUSSION THREAD

**Discussion by eestein** [rehypejs/rehype#104] [Q&A] [answered]
Is there a way to customize the generated href?
I'd like to remove the hash and add a default prefix to every link.

**Comment by wooorm** [comment, marked as answer]
Remove the hash? That's all there is!

Adding a prefix could be a PR.

**Reply by eestein** [reply]
lol
remove *AND* add another prefix to it :D

In any case, I'll take a look at the code and implement my own local version.
It'd be nice to have some customization options, though, like the icon classes and being able to customize the URL of the href

Thanks.

**Reply by wooorm** [reply]
Sounds like you want to create your own hrefs. In that case, I recommend doing what you want to do yourself.

Much of unified uses one type of IDs, in the style of GitHub, which sets an enormous precedent. That makes everything easier. I personally don’t see any reason to use a different “hashing” algorithm.
One good way trumps many different, potentially crappy, ways.

**Comment by pReya** [comment]
I'm also very interested in this feature. I would love to turn my links into absolute links by adding a URL as a prefix. The reason for this is that I am using a `base` tag on my website, which breaks anchor links.

**Reply by wooorm** [reply]
refs https://github.com/rehypejs/rehype-autolink-headings/pull/16
